CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+All contributions are **welcome** and **very much appreciated**.
+
+We accept contributions via Pull Requests on [Github](https://github.com/trikoder/oauth2-bundle).
+
+## Pull Request guidelines
+
+- **Add tests!** - If you are contributing with code, your contribution won't be accepted if it doesn't have tests.
```

Comment:
@toniperic Even though I completely agree with this, it just sounds a bit too harsh. We don't want to drive away potential contributors. Perhaps something in the line of "it is important to add tests that confirm your changes work as expected". I know this leaves a huge door open for people skipping on writing tests, but in my experience whoever is willing to create a PR is usually willing to do it right, which includes tests :smile: